src/app/sign-up-metamask/sign-up-metamask.component.html

```diff
@@ -0,0 +1,98 @@
+<app-banner></app-banner>
+
+<div
+  class="container home-container text-center"
+  *ngIf="globalVars.inTab || globalVars.webview"
+>
+  <div *ngIf="this.currentScreen === SCREEN.CREATE_ACCOUNT">
+    <div class="mb-20px">
+      <span class="title-text">Sign up / Log in using MetaMask</span>
+      <div class="main-text mb-2">
+        Enter the Decentralized Social using your MetaMask wallet.
+      </div>
+      <p class="txtalj mt-20px">
+        DeSo is a novel blockchain built for web3 social networks. It enables a
+        new category of apps that combine the innovations of decentralized
+        finance and social. To get started, let's set you up with an account.
+        This Metamask integration will allow you to create a DeSo wallet derived
+        off of your ETH wallet. When you click the button below, you will be
+        asked to
+        <a
```

Comment:
will clicking this redirect users? we probably don't want that. Let's open a new window or something else so they stay in their current place in the flow.

Reply:
It returns them to the app they had open which I think makes sense

Reply:
Open in new tab